### PR TITLE
Add resizable panels

### PR DESCRIPTION
**Pull request by mrkoreye**
Make the code panels resizable

resizers

### CODE REVIEW

**Review by steve8708 — APPROVED** [review]
sweet!

File: packages/fiddle/src/components/Fiddle.tsx [hunk 1]
```diff
@@ -267,6 +271,25 @@ export default function Fiddle() {
     }
   });
 
+  useEventListener<MouseEvent>(document.body, 'mousemove', (e) => {
```
Comment (steve8708) [inline packages/fiddle/src/components/Fiddle.tsx:274]: super elegant

**Comment by steve8708** [comment]
two mini pieces of feedback
- would be nice to set a small minimum width/height for each so you can't totally wipe a panel out (e.g. you drag too far and now accidentally can't drag again)
- would be nice to sync the values in localstorage so your desired layout persists after reload (e.g. how we handle `options.reactStyleType` now)

**Comment by mrkoreye** [comment]
@steve8708 updated

**Review by steve8708 — APPROVED** [review]

**Comment by steve8708** [comment]
I love it

**Review by mandx — APPROVED** [review]

**Pull request merged**